app/pages/admin-dashboard.tsx

```diff
@@ -233,34 +233,39 @@ export default function AdminDashboard({ session }: PageProps) {
                 onChange: setSelectedEnvironments,
                 options: environmentOptions,
                 label: 'Environments',
+                dataTestId: 'filter-env',
```

Comment:
@kurokn why are you passing `dataTestId` from the parent component? with this approach, every parent need to pass their own version of`dataTestId`